README.md

```diff
@@ -34,39 +34,41 @@ Please visit [Project Website](https://abdownloadmanager.com) for more info
 
 ## Installation
 
+[Download Instructions](https://abdownloadmanager.com)
+
+[GitHub Releases](https://github.com/amir1376/ab-download-manager/releases/latest)
+
```

Comment:
I believe the previous Installation section was more compact and straightforward. Let's retain that version for now.

Reply:
I think the dropdown installation instruction is very small in the all Readme, I couldn't find the script at first sight